kitsune/messages/jinja2/messages/includes/macros.html

```diff
@@ -52,27 +52,27 @@
 
 {% macro outbox_message(message) -%}
   <span class="to">
-  {% if message.recipient %}
+  {% if message.to %}
```

Comment:
I don't know, but I'm wondering if it would be better to use `if message.recipients_count > 0` here?

Reply:
Changed.